scss/previews/_color.scss

```diff
@@ -5,35 +5,50 @@
 // Color Preview
 // -------------
 
-[data-sassdoc='color-palette'] {
-  margin: size('gutter') 0;
+.color-palette {
+  display: flex;
+  flex-wrap: wrap;
+  margin: negative('shim');
+
+  @supports (display: grid) {
```

Comment:
I've been wondering about this. If a browser doesn't support grid, it will ignore all grid-* properties so the only thing you really need is the margin in this @supports statement. I know we get an error message in our current lint config though when you use the same property twice. I think this won't matter much here, but the parts when we aren't declaring a new margin (or other non-grid property) then we could clean up a number of nested queries going forward. Thoughts on that topic in general (not specific to this example)?

Reply:
I agree. Wherever it's easy to do without the feature-query, we just do that. Here, since we need the query for one value, I thought it was cleaner to just keep them all together. I don't feel strongly about that.